src/main/java/seedu/duke/commands/DeleteCommand.java

```diff
@@ -17,19 +22,26 @@ public DeleteCommand(String[] command) {
 
     public CommandResult execute() {
 
-        String removedTaskTitle;
+        fillCombinedItemList();
 
+        String removedTaskTitle;
         removedTaskTitle = deleteTask(taskToDelete);
         return new CommandResult("This task has been removed: " + removedTaskTitle
             + System.lineSeparator());
     }
 
+    public static void fillCombinedItemList() {
```

Comment:
Maybe i can add a method, in parser to get the combined list, instead of having to type it out multiple times? Such as parser.combinedList() ?